charts/simple-app/templates/deployment.yaml

```diff
@@ -132,6 +132,9 @@ spec:
 
         {{- end }}
         {{- end }}
+        {{- if and .Values.datadog.enabled .Values.datadog.scrapeLogs.enabled }}
+        ad.datadoghq.com/{{ include "simple-app.name" . }}.logs: {{ include "simple-app.datadogScrapeLoggingConfig" . | toJson }}
+        {{- end }}
```

Comment:
```    
        prometheus.io/path: /metrics
        ad.datadoghq.com/simple-app.logs: "\n- source:simple-app\n  service:simple-app\n  log_processing_rules:[map[name:exclude_datadoghq_users pattern:\\\\w+@datadoghq.com type:exclude_at_match]]"
      labels:
```

Reply:
This doesn't look right ... the `[map[name:...` stuff looks like it isn't real JSON and will cause an issue. Also, can we make this easier to read by using yaml pipes to make this print out as multiline json?

eg instead of:
```yaml
ad.datadoghq.com/simple-app.logs: "\n- source:simple-app\n  ...
```

do
```yaml
ad.datadoghq.com/simple-app.logs: |-
  { "source": "simple-app",
    ...
  }
  ...
```